doc/source/dev/index.rst

```diff
@@ -2,6 +2,31 @@
 Contributing to NumPy
 #####################
 
+Not a coder? Not a problem! NumPy is multi-faceted, and we can use a lot of help.
+These are all activities we'd like to get help with (they're all important, so
+we list them in alphabetical order):
+
+- Code maintenance and development
+- Community coordination
+- DevOps
+- Developing educational content
```

Comment:
Stefan commented in the call that it wasn't clear that this included documentation writing. So need to change to something like:

- Developing educational content & narrative documentation
- Writing technical documentation